src/locale/bootstrap-table-es-CR.js

```diff
@@ -85,6 +85,15 @@ $.fn.bootstrapTable.locales['es-CR'] = {
   },
   formatAdvancedCloseButton () {
     return 'Close'
+  },
+  formatFilterControlSwitch () {
+    return 'Hide/Show controls'
```

Comment:
Return this: Ocultar/Mostrar controles.

I think this can be applied for all Spanish translations

Reply:
Added it for all Spanish locale files.